src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -1146,6 +1146,72 @@ class TcgStatics {
   static boolean isGxPerformed(){
     bg.em().retrieveObject("gx_"+my.owner)
   }
+
+  /* General checks for attacks and abilities */
+
+  //  * assertBench
+  //    - Can be used with no arguments, or with a condition and additional text included.
+  //    - If a closure is given, it'll check for any "it" element in bench returning true.
+  //    - Optional params:
+  //      + isType: if set, restricts to benched Pokémon of a single specific type.
+  //      + hasPokemonEX/hasPokemonGX/hasPokemonV: Can be expanded if needed. All of these unset will have the method search for any Pokémon no matter what, but if even a single one is set true it'll only filter those that are set true as well.
+  //      + opp: If true, checks for the opponent's bench instead of "my" bench.
+  //      + repText: If true, cText will override the entire fail message instead of being added at the end.
```

Comment:
Please use JavaDoc / GroovyDoc style to write comments for methods

Reply:
If you use an IDE they usually autogenerate it when you type `/**` and press return.

Reply:
Tried to update this, tho I'm not sure if it follows GroovyDocs 100% in the params section.